qiskit/transpiler/_transpiler.py

```diff
@@ -133,6 +133,8 @@ def compile(circuits, backend,
             'register_slots': sum(register.size for register
                                   in circuit.get_cregs().values())})
         experiment.config = QobjItem(**experiment_config)
+        if seed:
```

Comment:
Actually there was - [`test/python/test_skip_transpiler.py::CompileSkipTranslationTest::test_simple_execute`](https://github.com/Qiskit/qiskit-terra/blob/master/test/python/test_skip_transpiler.py#L40) is failing for the Python simulator if the `seed` is not present on the experiment-level config with:
```
AssertionError: {'01': 539, '00': 485} != {'01': 533, '00': 491}
```

as it seems it is not using a seed. It could be an issue with the changes in the simulator (or the test), though - I'll look into it.

> The qobj schema allows for different experiments in a qobj to have their own individual "experiment-level" seed, but that is not what is passed to this compile() method.

Actually  I was planning to discuss a bit how does `compile()` handles and should handle the experiment-level variables. By the spec, for a Qobj similar to:

```
{
id=...,
config={seed=1},
experiments=[
   {instructions=...,       # experiment A
    config={seed=2}},
  {instructions=...}        # experiment B
]}
```

* experiment A will be run with `seed=2`
* experiment B will be run with `seed=1` ("inherited")

however, what should happen if `compile()` is called with something similar to:
```
compile(circuits, ..., config={a: blah, seed: 3}, seed=4)
```
could you confirm which seed would have preference if both are specified?

Reply:
Upon closer inspection, I updated the simulator so it does take into account the qobj-level seed. Still, the question about `compile()` still stands!